### ISSUE TITLE
ios waver not accessible with voiceover screenreader for blind

### ISSUE THREAD

**Issue by denizsincar29** [ggerganov/ggwave#79]
hello. recently i installed waver on my iphone, and voiceover says (no items on this window). i am blind. screenreaders cannot read graphical elements without label or non-native graphical interface elements. please read about blind accessibility in general. it's not so difficult.
